src/components/Header/index.js

```diff
@@ -241,19 +257,43 @@ function Header() {
   const [elevation, setElevation] = useState(0);
   window.addEventListener("scroll", () => {
     if (window.scrollY > 0) {
-      setElevation(9);
+      setElevation(8);
     } else {
       setElevation(0);
     }
   });
 
+  //   window.addEventListener("scroll", changeBackground);
+  let bgcolor = "linear-gradient(90deg, #C1DFC4 0%, #DEECDD 100%);";
+  console.log(location);
+
+  window.addEventListener("scroll", () => {
+    let count = isActive ? 900 : 770;
+    if (window.scrollY > count) {
+      setposition(false);
+    } else {
+      setposition(true);
+    }
+  });
+  console.log(position, "postion");
+
   return (
     <ThemeProvider theme={theme}>
       <AppBar
-        sx={{ zIndex: 10000 }}
-        position="sticky"
-        color="background"
         elevation={elevation}
+        sx={
+          bgColor
+            ? {
+                background: position ? bgcolor : "background",
```

Comment:
Add the color value in bgcolor instead `"linear-gradient(90deg, #C1DFC4 0%, #DEECDD 100%)"`